File: check_user.go
// Random Go struct generated on 2025-08-21
package main

import (
    "encoding/json"
    "fmt"
    "time"
)

type ProcessResponse struct {
    Name      string    `json:"name"`
    Value     int       `json:"value"`
    Active    bool      `json:"active"`
    CreatedAt time.Time `json:"created_at"`
}

func NewProcessResponse(name string, value int) *ProcessResponse {
    return &ProcessResponse{
        Name:      name,
        Value:     value,
        Active:    true,
        CreatedAt: time.Now(),
    }
}

func (s *ProcessResponse) UpdateValue(newValue int) {
    s.Value = newValue
}

func (s *ProcessResponse) ToJSON() ([]byte, error) {
    return json.Marshal(s)
}

func main() {
    obj := NewProcessResponse("sample", 77)
    data, _ := obj.ToJSON()
    fmt.Println(string(data))
}

File: check_user.rb
# Random Ruby class generated on 2025-08-24

class ValidateResult
  attr_reader :name, :value, :created_at
  attr_writer :name, :value

  def initialize(name, value = 83)
    @name = name
    @value = value
    @created_at = Time.now
  end

  def to_hash
    {
      name: @name,
      value: @value,
      created_at: @created_at.iso8601
    }
  end

  def update_value(new_value)
    old_value = @value
    @value = new_value
    "Updated from #{old_value} to #{new_value}"
  end
end

# Example usage
obj = ValidateResult.new("sample", 18)
puts obj.to_hash
